### PR TITLE
https://issues.redhat.com/browse/ACM-5762 New hive behavior

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

File: clusters/cluster_lifecycle/cluster_pool_destroy.adoc [hunk 1]
```diff
@@ -13,8 +15,12 @@ To destroy a cluster pool, complete the following steps:
...
 
-. In the _Options_ menu for the cluster pool that you want to delete, select *Destroy cluster pool*. Any unclaimed clusters in the cluster pool are destroyed. It might take some time for all of the resources to be deleted, and the cluster pool remains visible in the console until all of the resources are deleted. 
+. In the _Options_ menu for the cluster pool that you want to delete, type `confirm` in the confirmation box and select *Destroy*.
++
+*Notes:*
++
+* The *Destroy* button is disabled if the cluster pool has any cluster claims.
 +
-The namespace that contained the ClusterPool will not be deleted. Deleting the namespace will destroy any clusters that have been claimed from the ClusterPool, since the ClusterClaim resources for these clusters are created in the same namespace.
+* The namespace that contains the cluster pool is not be deleted. Deleting the namespace destroys any clusters that have been claimed from the cluster pool, since the cluster claim resources for these clusters are created in the same namespace.
```
Comment (chenz4027) [inline clusters/cluster_lifecycle/cluster_pool_destroy.adoc]: @oafischer Can we do `is not be deleted`? Others look good.

Comment (oafischer) [reply clusters/cluster_lifecycle/cluster_pool_destroy.adoc]: Good catch, just fixed it!

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**